.chloggen/prw-flaky-test.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
I know I asked you to add this changelog entry, but I think the correct option is to add the `skip changelog` label

Reply:
Should I remove it, then?

Reply:
yes please